### DISCUSSION TITLE
Appearance setting to disable right-click/alt-click menu-bar toggle

### DISCUSSION THREAD

**Discussion by balupton** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#716] [Ideas]
Right now when right-clicking the menu bar it changes my charge limit back to 100% instead of 55% — yet I am in the habit of right clicking menu bar items to open their preferences menu, which is not the behaviour of AlDente. It would be nice if there was an option in the "Appearance" preferences to disable this right-click toggle behaviour.

**Comment by mr-kew** [comment]
You can pick between nothing, low power mode or top up now in 1.26.1. I think this discussion can be closed


<img width="514" alt="image" src="https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/assets/26764516/08791507-0e8b-4519-a371-3903e8eb68d3">


Edit: It's under Features tab, not Appearance
